### DISCUSSION TITLE
Extract id3 from files

### DISCUSSION THREAD

**Discussion by TheoCouss** [directus/directus#15475] [Feature Requests]
Hi,

Nowadays Directus, on each uploaded file will extract the metadata of the images! I would be great to have the same functionality on audio files!!

The workflow I have in mind:
When a file is uploaded to Directus -> (FileService) get id3 data like exif data in images > store as basic json in the "metadata" column

**Comment by loan-talvat** [comment]
That would be great !

**Reply by benhaynes** [reply]
I agree, we should capture relevant metadata from all popular file types. I wonder if some of this data should go into dedicated fields... or an open ended metadata JSON field.

**Reply by loan-talvat** [reply]
To keep it simple I sincerely think that a JSON like for images currently is very good

Otherwise it may pose a lot of conditions... an audio stock, a thumbnail, titles... an image, the ISO, the aperture...

**Comment by azrikahar** [comment]
#6865 could be a relevant PR here.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
